src/components/DataCollector/index.js

```diff
@@ -60,6 +61,8 @@ const createDataCollector = ({ config }) => {
       .then(() => makeServerCall(event));
   };
 
+  createClickActivityCollector(config, logger, createEventHandler);
```

Comment:
Maybe we should create the `Click Collector` if it's not enabled in the config? Which means you won't need to pass `config`?

Reply:
I don't think we want to attach an event handler if it's not enabled.

Reply:
:) Sorry I meant maybe we should NOT create.... not that important though.